app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -197,13 +207,7 @@ class MainActivity : ComponentActivity() {
             RecycleSearch("Android")
         }
 
-        Button(
-            onClick = {
-                signIn()
-            }
-        ) {
-            Text(text = "Logon")
-        }
```

Comment:
Removed the logon button since this was something discussed in the class lecture videos.  Users should not feel pressure to sign up for anything.  Once they've started using the app and find it useful they will be more willing to sign up for an account.  The button looked kind of out of place where you had it.  I would recommend having "logon" as one of your menu items when the iconButton is clicked.

Reply:
Without this button, is there any way for users to logon?